terraform/cluster/aws/main.tf

```diff
@@ -142,3 +142,45 @@ resource "aws_launch_template" "cluster" {
     }
   }
 }
+
+module "ebs_csi_driver_controller" {
+  source  = "DrFaust92/ebs-csi-driver/kubernetes"
+  version = "3.3.1"
+
+  ebs_csi_controller_image                   = "public.ecr.aws/ebs-csi-driver/aws-ebs-csi-driver"
+  ebs_csi_driver_version                     = var.arm_type ? "v1.9.0-linux-arm64-amazon" : "v1.9.0"
+  ebs_csi_controller_role_name               = "convox-ebs-csi-driver-controller"
+  ebs_csi_controller_role_policy_name_prefix = "convox-ebs-csi-driver-policy"
+  oidc_url                                   = aws_iam_openid_connect_provider.cluster.url
+}
+
+resource "kubernetes_storage_class" "default" {
+  metadata {
+    name = "gp3"
+    annotations = {
+      "storageclass.kubernetes.io/is-default-class" = "true"
+    }
+  }
+
+  storage_provisioner    = "ebs.csi.aws.com"
+  volume_binding_mode    = "WaitForFirstConsumer"
+  allow_volume_expansion = false
```

Comment:
it might be a good idea to have this one as `true` to be more flexible and allow volumes to be expanded.